src/main/java/com/denys/toukdemo/dto/TicketType.java

```diff
@@ -0,0 +1,18 @@
+package com.denys.toukdemo.dto;
+
+public enum TicketType {
+
+    ADULT(25.0),
```

Comment:
a jak przyjdzie do nas biznes z nowym cennikiem to przekompilujemy projekt?

Reply:
> a jak przyjdzie do nas biznes z nowym cennikiem to przekompilujemy projekt?

można dodać możliwość wskazania cenników w bazie lub z pliku konfiguracyjnego. W tym momencie tak, trzeba będzie przekompilować projekt